usermods/PIR_sensor_switch/PIR_sensor_switch.cpp

```diff
@@ -317,7 +317,15 @@ void PIRsensorSwitch::publishHomeAssistantAutodiscovery()
     device[F("mf")]   = F(WLED_BRAND);
     device[F("mdl")]  = F(WLED_PRODUCT_NAME);
     device[F("sw")]   = versionString;
-    
+
+    String mqtt_avail_topic = mqttDeviceTopic;
+    mqtt_avail_topic += F("/status");

```

Comment:
Please add comments what these do. I.e. add a link to HA (https://www.home-assistant.io/integrations/mqtt/#sensors)